Routes/address.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const {
+	getAddress,
+	addAddress,
+	updateAddress,
+	deleteAddress,
+} = require('../Controllers/address');
+const router = express.Router();
+const { getAddressById, isAuthorized } = require('../Controllers/param');
+
+router.get('/address', isAuthorized, getAddressById, getAddress);
+router.post('/address', isAuthorized, getAddressById, addAddress);
+router.post('/address/:addressId', isAuthorized, getAddressById, updateAddress);
+router.delete(
+	'/address/:addressId',
+	isAuthorized,
```

Comment:
Instead of adding middleware in each route, you can simply write 
```
router.use(isAuthorized)
```
before all the routes using this middleware